README.md

```diff
@@ -1,58 +1,66 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Microverse CAPSTONE I
+
+## CAPSTONE I 
+
+![screenshot](./screen_apresent.png)
+
+## CAPSTONE I (Mobile screenshots)
+
+![screenshot](./screen_mobile.png)
 
-> One paragraph statement about the project.
-![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JS
+- Bootstrap 5
+
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+- [Live Demo](https://iwillteixeira.github.io/microverse-capstone/index.html)
+- [Video](https://iwillteixeira.github.io/microverse-capstone/index.html)
```

Comment:
- **Kindly provide a video link that works because this link sends you to the live demo project**

Reply:
Hello Carlos!

Thanks for reviewing my project, I just fix the README.md with the right link to the video.

With gratitude Will.